File: aoc2018/src/all.js
console.log("🎄 Advent of Code 2018\n");
for (let d = 1; d <= 9; d += 1) {
    console.log(`> Day ${d}`);
    const day = d >= 10 ? d : `0${d}`;
    require(`./day${day}.js`);
    console.log();
}

File: aoc2019/src/all.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    cout << "Advent of Code 2019\n" << endl;
    for(int d = 1; d <= 14; d++) {
        cout << "> Day " << d << endl;
        string day = (d < 10) ? "0" + to_string(d) : to_string(d);
        string command = "g++ src/day" + day + ".cpp -o day";
        system(command.c_str());
        system("./day");
        cout << endl;
    }
}
